### DISCUSSION TITLE
Add context classes to .private-view

### DISCUSSION THREAD

**Discussion by Guiqft** [directus/directus#5959] [Feature Requests]
Like we has for the `collections-overview`, adding classes for all the contexts like: `settings, files items, users` would help users to target custom css's on just a limited scope. 
This idea surged on a discussion about hidind the `.module-nav` div just on a certain module and maintain at the rest of them.

**Comment by philleepflorence** [comment]
### Route Context Options
- Add class to `.private-view` element - `settings`, `files`, `collections`, `items`, or `item` in [Private View Component](https://github.com/directus/directus/blob/main/app/src/views/private/private-view.vue) for layout context.
- Add URL slug to the `#app` element - `data-url-path` in [Main App Component](https://github.com/directus/directus/blob/main/app/src/app.vue) for granular context.

I have those options as planned code extensions to satisfy some requests for D8 to D9.
Happy to implement this sooner if needed.

#### Possible Related Discussions
[How to use custom css in project setting? #3902](https://github.com/directus/directus/discussions/3902)
[Project Settings: Custom CSS/JS Override Fields #3096](https://github.com/directus/directus/discussions/3096)

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
